src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -0,0 +1,48 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.model.ShoppingCart;
+import mate.academy.spring.model.dto.response.OrderResponseDto;
+import mate.academy.spring.service.OrderService;
+import mate.academy.spring.service.ShoppingCartService;
+import mate.academy.spring.service.UserService;
+import mate.academy.spring.service.dto.mapping.impl.response.OrderResponseMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+    private final ShoppingCartService shoppingCartService;
+    private final OrderService orderService;
+    private final UserService userService;
+    private final OrderResponseMapper orderResponseMapper;
```

Comment:
use abstraction, not implementation
```suggestion
    private final DtoResponseMapper<OrderResponseDto, Order> orderResponseMapper;
```